app/views/users/show.html.erb

```diff
@@ -1,12 +1,20 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Posts</title>
-  </head>
-  <body>
-    <h1>Here is a selected user from the list.</h1>
-  </body>
-</html>
\ No newline at end of file
+<div id="user-container">
+  <%= render partial: "users/user_info", locals: { user: @user } %>
+</div>
+
+<div id="user-biography">
+  <h3>Biography</h3>
+  <p><%= @user.bio %></p>
+</div>
+
+<div id="post-container">
+  <% @user.three_most_recent_posts.each_with_index do |post, i| %>
+    <li>
+      <%= link_to "/users/#{@user.id}/posts/#{post.id}", id: "post" do %>
+        <%= render partial: "posts/post_info", locals: { post: post } %>
+      <% end %>
+    </li>
+  <% end %>
+</div>
+
+<%= link_to "See all posts", "/users/#{@user.id}/posts?page=1" %>
```

Comment:
- [ ] The "Comments: 1, Likes: 0" text should be aligned to the right of the div like in the design.

- [ ] The "See all posts" button should be centered horizontally as in the design.

  ![image](https://github.com/katarighe/blog-app/assets/81468675/05ba7a13-7536-4c68-a634-2617a8da9bc9)

  ![image](https://github.com/katarighe/blog-app/assets/81468675/437066a6-96ad-41cc-9f06-a6e8561baa7a)